config.schema.json

```diff
@@ -218,11 +219,40 @@
       "$ref": "#/$defs/additional_ibc_token_info"
     }
   },
+  "demex_trading_league_config": {
+    "type": "object",
+    "description": "Config for Demex Trading League",
+    "properties": {
+      "type": "object",
+      "promoMarkets": {
+        "type": "array",
+        "description": "List of market names designated as Promo markets",
```

Comment:
`List of market names designated as promo markets during the Demex Trading League competition`